packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -95,6 +114,10 @@ export class EpochTracker implements IPersistedFileCache {
 
     public async put(entry: IEntry, value: any) {
         assert(this._fluidEpoch !== undefined, 0x1dd /* "no epoch" */);
+        // For snapshots, the value should have the cache entry time
```

Comment:
nit: consider adding that this will be used to determine the age of the entry so that the entry can be expired.